### DISCUSSION TITLE
Strip html tags off affected entities

### DISCUSSION THREAD

**Discussion by D4RKMATT3R** [GhostManager/Ghostwriter#283] [Q&A] [answered]
Hi, just wanted to ask if it is possible to strip the html tags off of the affected entities, since I want that section in my report to preserve the font type I have selected . strip_html does not seem to work here.

**Comment by D4RKMATT3R** [comment, marked as answer]
The Question asnwered itself, strip_html works as it is supposed to I must have done something wrong.

**Comment by chrismaddalena** [comment]
Thanks for updating the question!
